src/js/toolbar.js

```diff
@@ -356,6 +356,7 @@ var Toolbar;
         showAndUpdateToolbar: function () {
             this.modifySelection();
             this.setToolbarButtonStates();
+            this.base.trigger('updateToolbar', {}, this.base.getFocusedElement());
```

Comment:
naming things is hard but that's superoverly verbose, so -1 imo. I can add a test or two, but can't get to it today or tomorrow for sure. I only merged this over because I *believe* it to be the last monkey patch I've been maintaining locally and am attempting to forward port all of our internal app usage to newest editor (post extension and cleanup stuff, which btw thanks for hitting all of our talking points mostly, with .document and .base/.parent et al).  If this won't be merged I'll just extension-in a new version of this 5-line function to get past last of deprecation hurdles and double back on tests and this MR later in the week.